app/Mail/EmailSender.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Mail;
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+use PHPMailer\PHPMailer\SMTP;
+
+class EmailSender {
+
+    private $mailer;
+
+    public function __construct()
```

Comment:
on each instance of the class you gonna to create a new phpmailer service? maybe you need to implement singleton here?